pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -130,13 +132,17 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 				{
 					APIGroups: []string{coordinationv1.GroupName},
 					Resources: []string{"leases"},
-					Verbs:     []string{"create"},
+					Verbs:     []string{"get", "list", "patch", "update", "watch", "create", "delete"},
```

Comment:
Why was the lease part changed? I think what was there before is pretty much standard - why is this different now?

Reply:
Earlier the permission was for only one lease. But now we need permission for more leases